kafka-rest/src/test/java/io/confluent/kafkarest/integration/ClusterTestHarness.java

```diff
@@ -404,6 +404,23 @@ protected final void createTopic(String topicName, int numPartitions, short repl
     }
   }
 
+  protected final void createTopic(
```

Comment:
So using the older way to createTopic by specifying the partitions and replicationFactor, it was not known what partitions the broker would be leader for, is that right ?

Reply:
Yes. This way we can be sure.